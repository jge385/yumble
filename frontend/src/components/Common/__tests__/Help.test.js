import React from 'react';
import renderer from 'react-test-renderer';
import Help from '../Help';

test('Location Helper snapshot testing with on/off trigger on', () => {
  const component1 = renderer.create(
      <>
        <Help trigger={true}/>
      </>,
  );

  const tree1 = component1.toJSON();
  expect(tree1).toMatchSnapshot();
});

test('Location Helper snapshot testing with on/off trigger off', () => {
  const component2 = renderer.create(
      <>
        <Help trigger={false}/>
      </>,
  );

  const tree2 = component2.toJSON();
  expect(tree2).toMatchSnapshot();
});

