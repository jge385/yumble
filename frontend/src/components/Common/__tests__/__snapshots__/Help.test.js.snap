// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Location Helper snapshot testing with on/off trigger off 1`] = `""`;

exports[`Location Helper snapshot testing with on/off trigger on 1`] = `
<div
  className="Help"
>
  <div
    className="Help-inner"
  >
    <button
      className="SmallBtn"
      id="CloseButton"
      onClick={[Function]}
    >
      close
    </button>
  </div>
</div>
`;
